### ISSUE TITLE
Lokinet will not connect to exit nodes on windows 10 21H1

### ISSUE THREAD

**Issue by Destroyinator69420** [oxen-io/lokinet#1749]
**Describe the bug**
Lokinet will connect to Snapps like cafe.loki, however, it will not connect lokinet exit nodes. I have used exit.loki for my test.

**To Reproduce**
1. Upgrade lokinet 0.9.5 to Lokinet 0.9.6.
2. Check to make sure that tuntap is installed.
3. Run lokinet as administrator. (It will not run any other way)
4. Before turning lokinet on, enter exit.loki into the exit node box, and then flip the switch
5. Turn lokinet on
6. Go to iplocation.net in a Firefox private tab on version 92
7. Notice how the IP address did not change

Windows details:
Edition	Windows 10 Home
Version	21H1
Installed on	‎10/‎30/‎2020
OS build	19043.1237
Experience	Windows Feature Experience Pack 120.2212.3530.0

**Screenshots Or Logs**
I will not post a screenshot due to metadata security concerns

**Device and Operating system (please complete the following information):**
 Lenovo c940 running windows 10 21H1

**Comment by jagerman** [comment]
Does it work if you start lokinet first, verify that it is operational, and *then* flip the switch to enable the exit?

**Comment by Destroyinator69420** [comment]
@jagerman Let me try that.

**Comment by Destroyinator69420** [comment]
Doesn't work

**Comment by GrahamboJangles** [comment]
I've been trying to get Lokinet working on Windows for some time now. Since version 0.8.5, the furthest I've got was by running a batch script in `Lokinet/bin` that runs `lokinet-vpn.exe --up --exit tixe.loki` (among other exit nodes, that one had the best latency for me) but they only worked for about a minute and then I lost connection to Lokinet and had to `lokinet-vpn.exe --down` and 
`lokinet-vpn.exe --kill` then restart the script. I've now tried this script in 0.9.7 and now I get `failed to connect to lokinet RPC: connection attempt timed out` when trying to connect to any exit node, or even when I try to run either `lokinet-vpn.exe --down` or `lokinet-vpn.exe --kill`.

Do I have to run `tuntap-install.exe` that is in `Lokinet/bin`?

Here is the verbose response:
```
A:\Lokinet\bin>lokinet-vpn.exe --verbose --up --exit tixe.loki
debug [oxenmq/oxenmq.cpp:212] generating x25519 keypair for remote-only OxenMQ instance
info [oxenmq/oxenmq.cpp:241] Initializing OxenMQ remote-only with pubkey 38491db62cb1f2ea0585ac4b5e58baa833338de34e842625c73fb7f98967d846
debug [oxenmq/oxenmq.cpp:255] Waiting for proxy thread to get ready...
debug [oxenmq/proxy.cpp:346] Reserving space for 13 max workers = 8 general plus reservations for:
debug [oxenmq/proxy.cpp:349]     - (batch jobs): 4
debug [oxenmq/proxy.cpp:350]     - (reply jobs): 1
debug [oxenmq/proxy.cpp:351] Plus 0 tagged worker threads
debug [oxenmq/oxenmq.cpp:267] Proxy thread is ready
debug [oxenmq/connections.cpp:323] Establishing remote connection to tcp://127.0.0.1:1190 (NULL auth)
debug [oxenmq/connections.cpp:340] Opened new zmq socket to tcp://127.0.0.1:1190, conn_id 1; sending HI
debug [oxenmq/connections.cpp:215] Closing conn index 0
debug [oxenmq/worker.cpp:65] New worker thread w0 (w0) started
failed to connect to lokinet RPC: connection attempt timed out
info [oxenmq/oxenmq.cpp:419] OxenMQ shutting down proxy thread
debug [oxenmq/worker.cpp:22] worker w0 waiting for RUN
debug [oxenmq/worker.cpp:36] Worker w0 received QUIT command, shutting down
debug [oxenmq/worker.cpp:265] Worker w0 exited normally
debug [oxenmq/proxy.cpp:23] Received quit command, shutting down proxy thread
debug [oxenmq/proxy.cpp:41] Proxy thread teardown complete
info [oxenmq/oxenmq.cpp:422] OxenMQ proxy thread has stopped

A:\Lokinet\bin>if 1 == 1 (lokinet-vpn.exe --verbose --up --exit exit.loki )
debug [oxenmq/oxenmq.cpp:212] generating x25519 keypair for remote-only OxenMQ instance
info [oxenmq/oxenmq.cpp:241] Initializing OxenMQ remote-only with pubkey a7452a9f1af5553fd0575e85ef68a954fc52788ff84325dfff6918bbc4508f54
debug [oxenmq/oxenmq.cpp:255] Waiting for proxy thread to get ready...
debug [oxenmq/proxy.cpp:346] Reserving space for 13 max workers = 8 general plus reservations for:
debug [oxenmq/proxy.cpp:349]     - (batch jobs): 4
debug [oxenmq/proxy.cpp:350]     - (reply jobs): 1
debug [oxenmq/proxy.cpp:351] Plus 0 tagged worker threads
debug [oxenmq/oxenmq.cpp:267] Proxy thread is ready
debug [oxenmq/connections.cpp:323] Establishing remote connection to tcp://127.0.0.1:1190 (NULL auth)
debug [oxenmq/connections.cpp:340] Opened new zmq socket to tcp://127.0.0.1:1190, conn_id 1; sending HI
debug [oxenmq/connections.cpp:215] Closing conn index 0
debug [oxenmq/worker.cpp:65] New worker thread w0 (w0) started
failed to connect to lokinet RPC: connection attempt timed out
info [oxenmq/oxenmq.cpp:419] OxenMQ shutting down proxy thread
debug [oxenmq/worker.cpp:22] worker w0 waiting for RUN
debug [oxenmq/worker.cpp:36] Worker w0 received QUIT command, shutting down
debug [oxenmq/worker.cpp:265] Worker w0 exited normally
debug [oxenmq/proxy.cpp:23] Received quit command, shutting down proxy thread
debug [oxenmq/proxy.cpp:41] Proxy thread teardown complete
info [oxenmq/oxenmq.cpp:422] OxenMQ proxy thread has stopped

A:\Lokinet\bin>if 1 == 1 (goto exit )

A:\Lokinet\bin>lokinet-vpn.exe --verbose --down
debug [oxenmq/oxenmq.cpp:212] generating x25519 keypair for remote-only OxenMQ instance
info [oxenmq/oxenmq.cpp:241] Initializing OxenMQ remote-only with pubkey c022b531df45e41cfa7d2215aa6d44fed887a34d433732573b7e3cc8e8f89971
debug [oxenmq/oxenmq.cpp:255] Waiting for proxy thread to get ready...
debug [oxenmq/proxy.cpp:346] Reserving space for 13 max workers = 8 general plus reservations for:
debug [oxenmq/proxy.cpp:349]     - (batch jobs): 4
debug [oxenmq/proxy.cpp:350]     - (reply jobs): 1
debug [oxenmq/proxy.cpp:351] Plus 0 tagged worker threads
debug [oxenmq/oxenmq.cpp:267] Proxy thread is ready
debug [oxenmq/connections.cpp:323] Establishing remote connection to tcp://127.0.0.1:1190 (NULL auth)
debug [oxenmq/connections.cpp:340] Opened new zmq socket to tcp://127.0.0.1:1190, conn_id 1; sending HI
debug [oxenmq/connections.cpp:215] Closing conn index 0
debug [oxenmq/worker.cpp:65] New worker thread w0 (w0) started
failed to connect to lokinet RPC: connection attempt timed out
info [oxenmq/oxenmq.cpp:419] OxenMQ shutting down proxy thread
debug [oxenmq/worker.cpp:22] worker w0 waiting for RUN
debug [oxenmq/worker.cpp:36] Worker w0 received QUIT command, shutting down
debug [oxenmq/worker.cpp:265] Worker w0 exited normally
debug [oxenmq/proxy.cpp:23] Received quit command, shutting down proxy thread
debug [oxenmq/proxy.cpp:41] Proxy thread teardown complete
info [oxenmq/oxenmq.cpp:422] OxenMQ proxy thread has stopped

A:\Lokinet\bin>lokinet-vpn.exe --verbose --kill
debug [oxenmq/oxenmq.cpp:212] generating x25519 keypair for remote-only OxenMQ instance
info [oxenmq/oxenmq.cpp:241] Initializing OxenMQ remote-only with pubkey 63d9c799bb9f0a8193dd32b2daa72602013d8a9f84e0658524ae59e2c42b1f14
debug [oxenmq/oxenmq.cpp:255] Waiting for proxy thread to get ready...
debug [oxenmq/proxy.cpp:346] Reserving space for 13 max workers = 8 general plus reservations for:
debug [oxenmq/proxy.cpp:349]     - (batch jobs): 4
debug [oxenmq/proxy.cpp:350]     - (reply jobs): 1
debug [oxenmq/proxy.cpp:351] Plus 0 tagged worker threads
debug [oxenmq/oxenmq.cpp:267] Proxy thread is ready
debug [oxenmq/connections.cpp:323] Establishing remote connection to tcp://127.0.0.1:1190 (NULL auth)
debug [oxenmq/connections.cpp:340] Opened new zmq socket to tcp://127.0.0.1:1190, conn_id 1; sending HI
debug [oxenmq/connections.cpp:215] Closing conn index 0
debug [oxenmq/worker.cpp:65] New worker thread w0 (w0) started
failed to connect to lokinet RPC: connection attempt timed out
info [oxenmq/oxenmq.cpp:419] OxenMQ shutting down proxy thread
debug [oxenmq/worker.cpp:22] worker w0 waiting for RUN
debug [oxenmq/worker.cpp:36] Worker w0 received QUIT command, shutting down
debug [oxenmq/worker.cpp:265] Worker w0 exited normally
debug [oxenmq/proxy.cpp:23] Received quit command, shutting down proxy thread
debug [oxenmq/proxy.cpp:41] Proxy thread teardown complete
info [oxenmq/oxenmq.cpp:422] OxenMQ proxy thread has stopped
```

**Comment by majestrate** [comment]
you need to have lokinet service running: `net start lokinet`

**Comment by GrahamboJangles** [comment]
> you need to have lokinet service running: `net start lokinet`

Thanks 👍
Doing great work!

**Comment by majestrate** [comment]
fixed via #1899
